src/styles/index.ts

```diff
@@ -4499,6 +4499,10 @@ const styles = (theme: ThemeColors) =>
             ...spacing.mr3,
         },
 
+        horizontalRule: {
+            borderColor: theme.border,
+        },
+
```

Comment:
Can we just add this style directly to the elements instead of creating a new style for the borderColor only?

Reply:
Not sure what you mean. Can't we do something like this? https://github.com/Expensify/App/blob/ea6f6a064e2a864d1802bbfa5cfcc71bba01b537/src/components/SelectionList/Search/TaskListItemRow.tsx#L94

Reply:
Ah I missed that! Good call! Updated!